### PR TITLE
Fix merchant dropdown for >100 options

### PR DESCRIPTION
**Pull request by ashwin1111**
https://fylein.slack.com/archives/C044R3CLT7F/p1665399377098549?thread_ts=1665380531.818199&cid=C044R3CLT7F

### CODE REVIEW

**Review by Shwetabhk — APPROVED** [review]

File: fyle_slack_app/slack/interactives/block_suggestion_handlers.py [hunk 1]
```diff
@@ -310,6 +310,8 @@ def handle_merchant_suggestion(self, slack_payload: Dict, user_id: str, team_id:
                 # In this case, this user entered text will get stored as a new merchant in merchants table
                 suggested_merchants = [merchant_value_entered]
 
+        # Trimming merchant options to have an upper limit of 100 options
+        suggested_merchants = suggested_merchants[0:100]
```
Comment (jatzz10) [inline fyle_slack_app/slack/interactives/block_suggestion_handlers.py:314]: @ashwin1111 do we keep an upper limit for this in web-app?

**Review by jatzz10 — COMMENTED** [review]
added a comment - please check @ashwin1111

File: fyle_slack_app/slack/interactives/block_suggestion_handlers.py (hunk 1 above)
Comment (ashwin1111) [reply fyle_slack_app/slack/interactives/block_suggestion_handlers.py:314]: no, we don't have upper limit

Comment (jatzz10) [reply fyle_slack_app/slack/interactives/block_suggestion_handlers.py:314]: got clarified [here](https://fylein.slack.com/archives/C044R3CLT7F/p1665473619650369)

**Review by jatzz10 — APPROVED** [review]

**Pull request merged**